search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -37,17 +35,52 @@ function onClickCountry(e) {
 function setCountryTrends(countryCode) {
   const topTrendsTab = document.getElementById('top-trends-tab');
   topTrendsTab.innerHTML = '';
-
-  fetch('/country-trends/' + countryCode).then(countryTrends => 
+  fetch('/country-trends/' + countryCode).then(countryTrends =>
       countryTrends.json()).then(trends => {
-    if (trends.length === 0) {
-      topTrendsTab.innerText = 'Trends are not available for the selected country.';
-    } else {
-      trends.forEach(trend => {
-        const trendHeader = document.createElement('h5');
-        trendHeader.innerText = trend.topic;
-        topTrendsTab.appendChild(trendHeader);
+        if (trends.length === 0) {
+          topTrendsTab.innerText = 
+              'Trends are not available for the selected country.';
+        } else {
+          trends.forEach(trend => {
+            const trendHeader = document.createElement('h5');
+            trendHeader.innerText = trend.topic;
+            topTrendsTab.appendChild(trendHeader);
```

Comment:
This is from @ChenyuHeidiZhang's code in the previous branch. I believe the reason why is that we want to add a feature so that when you click on the trend it expands to show the top search results for that trend, and so we wanted to larger? Chenyu can confirm.

Reply:
Yes Carmen's right